### ISSUE TITLE
Docs for icons missing accessibility feature

### ISSUE THREAD

**Issue by barrymcgee** [canonical-web-and-design/vanilla-framework#2573]
For accessibility purposes, you can add text within an icon that will not be displayed.

e.g. 

`<i className="p-icon--contextual-menu">This text will not be displayed</i>`

This feature is not documented here: https://docs.vanillaframework.io/patterns/icons/

**Comment by anthonydillon** [comment]
Thanks @barrymcgee great catch!
